From 6a8d65139e99915ffbcefeb79bf1dbc39cce938f Mon Sep 17 00:00:00 2001
From: "yiyang.wang" <yiyang.wang@intel.com>
Date: Thu, 8 Jul 2021 09:25:59 +0800
Subject: [PATCH 18/18] kconfig: aic: ip6table m -> y

some game require connecting server through vpn,
change the ip6table from kernel module to build in to satisfy vpn.

jira: https://jira.devtools.intel.com/browse/VVP0-3397

Signed-off-by: yiyang.wang <yiyang.wang@intel.com>
---
 arch/x86/configs/x86_64_sg1_defconfig | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/x86/configs/x86_64_sg1_defconfig b/arch/x86/configs/x86_64_sg1_defconfig
index de8f85c..a77c4c8 100644
--- a/arch/x86/configs/x86_64_sg1_defconfig
+++ b/arch/x86/configs/x86_64_sg1_defconfig
@@ -1134,7 +1134,7 @@ CONFIG_NETFILTER_NETLINK_ACCT=m
 CONFIG_NETFILTER_NETLINK_QUEUE=m
 CONFIG_NETFILTER_NETLINK_LOG=m
 CONFIG_NETFILTER_NETLINK_OSF=m
-CONFIG_NF_CONNTRACK=m
+CONFIG_NF_CONNTRACK=y
 CONFIG_NF_LOG_COMMON=m
 CONFIG_NF_LOG_NETDEV=m
 CONFIG_NETFILTER_CONNCOUNT=m
@@ -1165,7 +1165,7 @@ CONFIG_NF_CT_NETLINK=m
 CONFIG_NF_CT_NETLINK_TIMEOUT=m
 CONFIG_NF_CT_NETLINK_HELPER=m
 CONFIG_NETFILTER_NETLINK_GLUE_CT=y
-CONFIG_NF_NAT=m
+CONFIG_NF_NAT=y
 CONFIG_NF_NAT_NEEDED=y
 CONFIG_NF_NAT_PROTO_DCCP=y
 CONFIG_NF_NAT_PROTO_UDPLITE=y
@@ -1405,9 +1405,9 @@ CONFIG_NF_TPROXY_IPV6=m
 CONFIG_NF_DUP_IPV6=m
 CONFIG_NF_REJECT_IPV6=m
 CONFIG_NF_LOG_IPV6=m
-CONFIG_NF_NAT_IPV6=m
+CONFIG_NF_NAT_IPV6=y
 CONFIG_NF_NAT_MASQUERADE_IPV6=y
-CONFIG_IP6_NF_IPTABLES=m
+CONFIG_IP6_NF_IPTABLES=y
 CONFIG_IP6_NF_MATCH_AH=m
 CONFIG_IP6_NF_MATCH_EUI64=m
 CONFIG_IP6_NF_MATCH_FRAG=m
@@ -1419,13 +1419,13 @@ CONFIG_IP6_NF_MATCH_RPFILTER=m
 CONFIG_IP6_NF_MATCH_RT=m
 # CONFIG_IP6_NF_MATCH_SRH is not set
 CONFIG_IP6_NF_TARGET_HL=m
-CONFIG_IP6_NF_FILTER=m
+CONFIG_IP6_NF_FILTER=y
 CONFIG_IP6_NF_TARGET_REJECT=m
 CONFIG_IP6_NF_TARGET_SYNPROXY=m
-CONFIG_IP6_NF_MANGLE=m
-CONFIG_IP6_NF_RAW=m
+CONFIG_IP6_NF_MANGLE=y
+CONFIG_IP6_NF_RAW=y
 CONFIG_IP6_NF_SECURITY=m
-CONFIG_IP6_NF_NAT=m
+CONFIG_IP6_NF_NAT=y
 CONFIG_IP6_NF_TARGET_MASQUERADE=m
 CONFIG_IP6_NF_TARGET_NPT=m
 CONFIG_NF_DEFRAG_IPV6=m
-- 
1.8.3.1

